### ISSUE TITLE
Terraform 0.12

### ISSUE THREAD

**Issue by Trois-Six** [ciscoecosystem/terraform-provider-aci#33]
Hi,

I have a new problem: with Terraform 0.12, my run is blocked and never ends. If I execute terraform with TF_LOG=TRACE I see a lot a lines like 

```
...
2019/05/28 11:26:06 [TRACE] dag/walk: vertex "aci_l3_outside.XXX" is waiting for "aci_vrf.vrf_XXX"
2019/05/28 11:26:06 [TRACE] dag/walk: vertex "aci_bridge_domain.bd_YYY" is waiting for "aci_vrf.vrf_YYY"
...
2019/05/28 11:26:07 [TRACE] dag/walk: vertex "aci_application_epg.epg_XXX" is waiting for "aci_bridge_domain.bd_XXX"
...
```
Regards,

Pierre

**Comment by nkatarmal-crest** [comment]
Hi,

Can you please provide your .tf files that you are using?

Nirav.

**Comment by Trois-Six** [comment]
I just sent an email to Devarshi with the traces and tf files.

**Comment by nkatarmal-crest** [comment]
@Trois-Six i have tested the files on terraform 0.12 version i don't see any issue there.

**Comment by Trois-Six** [comment]
It must be related to the ACI version then...

**Comment by devarshishah3** [comment]
@Trois-Six , which APIC version are you using?

**Comment by Trois-Six** [comment]
3.0(2)n

**Comment by nkatarmal-crest** [comment]
@Trois-Six Is the issue still present there?

**Comment by Trois-Six** [comment]
Hi,
I just tried some minutes ago, and I have a problem with the API breakage in Terraform 0.12. I am using the latest version of the provider, from the "review" branch.
```
$ dpkg -l | grep terraform
ii  terraform                                                   0.12.3-1                                                          amd64        Tool for building, changing, and versioning infrastructure safely and efficiently
ii  terraform-provider-aci                                      0.0~git20190708.4a21600                                           amd64        Terraform aci provider
$ terraform plan

Error: Failed to instantiate provider "aci" to obtain schema: Incompatible API version with plugin. Plugin version: 4, Client versions: [5]
```
Pierre

**Comment by nkatarmal-crest** [comment]
@Trois-Six  This is a Terraform side bug. There is an issue in Terraform repository.https://github.com/hashicorp/terraform/issues/19221.

**Comment by carrajola** [comment]
Execute the following command:
terraform init -upgrade

**Comment by arian-saliu** [comment]
@nkatarmal-crest 

Hi There. We are experiencing some issues with our Cisco ACI.
Basically when we try to apply more than 10 resources using certificate authentication and with disabled throttling on Cisco ACI, also using parellelism=1 the terraform code runs forever.

For all of our resources we are using counts and at a certain point when more than 10 resources are being created the whole process starts running forever.

I tried building the provider from the review branch as well... but that didn't help either.

Please assist with that.

**Comment by nkatarmal-crest** [comment]
@arian-saliu I will look into it.

**Comment by nkatarmal-crest** [comment]
This issue is fixed and there is separate issue opened for #39. Closing this one.
